### DISCUSSION TITLE
@expo/env monorepo workspace root

### DISCUSSION THREAD

**Discussion by tyrauber** [expo/expo#24785] [Proposals and Ideas]
I like sharing my env files across multiple apps / packages in a pnpm monorepo. As a result I place my .env.* in workspace root.

I have found that I can source those files in my expo app, by manually calling load on @expo/env at the top of my metro.config with the workspaceRoot, and force: true as the options.

```
// metro.config.js
const { getDefaultConfig } = require('expo/metro-config');
const path = require('path');

const projectRoot = __dirname;
const workspaceRoot = path.resolve(projectRoot, '../..');

require('@expo/env').load(workspaceRoot, { force: true });
...
```

Is this advisable?  Is there a better option for workspace env variables?

**Comment by tsapeta** [comment]
It seems fine to me, but @byCedric might be a better person to answer this question.

**Reply by Silthus** [reply]
I am facing the same issue and even tried launching the app with expo-cli from the root and specifying the path `npx expo start apps/expo` but unfortunately it still only looks at the env files in the app dir.

I would expect that if I launch the expo cli from the root it would use that as a basis for the env.
